apps/src/lib/util/UserPreferences.js

```diff
@@ -162,4 +162,36 @@ export default class UserPreferences extends Record({userId: 'me'}) {
       return null;
     }
   }
+
+  async getGlobalTheme() {
+    try {
+      const themeResponse = await HttpClient.fetchJson(
+        '/user_preference/theme'
+      );
+      return themeResponse.value?.theme?.global;
+    } catch (error) {
+      // Don't log error if 'Not found', as it just means the
+      // user has not set a theme yet.
+      if (error.response.status !== 404) {
+        Lab2Registry.getInstance()
+          .getMetricsReporter()
+          .logError('Error fetching theme', undefined, {
+            message: error.response,
+          });
```

Comment:
minor: this assumes that the `getGlobalTheme` API will always be used by Lab2. do we want to make that assumption? we could also let the error get thrown if it's not a 404 and handle it within lab2?

Reply:
Good point, since it's only used for lab2 right now I'm inclined to keep it as-is and fast-follow with an improvement. Since this PR is pretty big on its own if it's not blocking I'd rather not do refactors that will slow it down, so we can get it in close to the launch date.